cellmlmanip/printer.py

```diff
@@ -342,14 +334,11 @@ def _print_Piecewise(self, expr):
             # Check if boolean True (if found, stop evaluating further)
             if isinstance(c, BooleanTrue):
                 other = self._print(e)
-                break
-            # Sympy filters these out:
-            # elif isinstance(c, BooleanFalse):
-            #    continue
-
-            # Add e-if-c-else-? statement
-            parts += self._print_ternary(c, e)
-            brackets += 1
+                # Sympy filters BooleanFalse out:
+            else:
```

Comment:
This is actually changing the logic: a piecewise with 3 cases where the middle one had `BooleanTrue` condition would now be handled differently.

Reply:
Not in practice as this would be filtered out (see below)